### PR TITLE
Added actuator for RGB-LEDs via FASTLED lib.

### PR DESCRIPTION
**Pull request by joen**
This is all a litte dirty, but working.
You could theoretically use all RGB-LEDs, that are supported by fastled. Just change the configuration according to config_FASTLED.h

I am not quite happy with the effects, which could be done in a more intelligent manner.
The overall idea is to have a color value send to a topic that is used as the color the effect is "played" in and as the color that is set, when the effect is finished.

### CODE REVIEW

**Comment by joen** [comment]
So i should fork the current version, integrate my changes an then issue a new pull request, right?

**Comment by 1technophile** [comment]
Yes it would be better

**Comment by 1technophile** [comment]
Hi @joen, do you think you would be able to do a new pull request ?

File: ZactuatorFASTLED.ino [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+      leds[i] = number;
+    }
+    FastLED.show();
+  } else if (topic == subjectMQTTtoFASTLEDscan){
+    for (int j = 10; j > 0; j--) {
+      for (int i = 0 ; i < FASTLED_NUM_LEDS; i++ ) {
+        leds[i] = number;
+        leds[i - 1] = CRGB::Black;
+        FastLED.show();
+        delay(100);
```
Comment (Legion2) [inline ZactuatorFASTLED.ino]: using delay will block the main loop.

**Comment by madl3x** [comment]
How is this going? Any chance we get to see it released in 0.9.x?
Thanks

**Comment by 1technophile** [comment]
v0.9.2 I don't think, could be the next if you are interested in testing it?

**Comment by BigDi** [comment]
I have a similar use case. I take the code from Joen and merge it into current dev branch. It will compile but I haven't tested it on my WEMOS.

**Comment by 1technophile** [comment]
This is great! Could you maybe submit a new updated PR?

**Comment by BigDi** [comment]
I created a new pull request. Hope every thing is corrent. Is my first one.
I have some trouble while testing the code with my WEMOS D1 mini. I get it only working if I used pin D2. Other pins are not working.

**Comment by joen** [comment]
closing, please refer to https://github.com/1technophile/OpenMQTTGateway/pull/433

**Pull request closed without merging**